### DISCUSSION TITLE
useSelect -  Update (native) HiddenSelect selected value

### DISCUSSION THREAD

**Discussion by mersudinh** [adobe/react-spectrum#3627] [General]
I am using useSelect with the native select element that is hidden(HiddenSelect). 
Is there a way to update the native select element when the option is selected and add a "selected" attribute?

<img width="982" alt="image" src="https://user-images.githubusercontent.com/56997983/195047119-fc06a3a9-0ae4-4558-a46e-1b861d792b99.png">

**Comment by snowystinger** [comment]
What reason do you need to add a "selected" attribute? Are you running into a problem with the component?

**Reply by mersudinh** [reply]
Not really a problem. It was a request that I have to implement somehow.

**Reply by snowystinger** [reply]
Can I ask why you need to implement it? What will it solve? It's a bit of an abstract request to me right now.
